<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>monitor_info_widget</class>
 <widget class="QWidget" name="monitor_info_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="main_vertical_layout">
    <property name="spacing">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="monitor_checkboxs_layout">
      <item>
       <widget class="QCheckBox" name="enabled_checkbox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="flipped_checkbox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Flipped</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="button_top_spacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="name_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="name_val_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="description_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="description_label">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="description_val_label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Unknown</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="resolution_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="resolution_label">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="resolution_combobox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="scale_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="scale_label">
        <property name="text">
         <string>Scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="scale_double_spinbox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="position_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="position_label">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontal_position_spinbox_layout">
        <item>
         <widget class="QSpinBox" name="position_x_spinbox">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="position_y_spinbox">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="maximum">
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="transform_horizontal_layout" stretch="1,4">
      <item>
       <widget class="QLabel" name="transform_label">
        <property name="text">
         <string>Transform</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="transform_combobox">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
